site/en/bazel.css

```diff
@@ -136,3 +136,8 @@
   margin-inline-start: 8px;
   vertical-align: text-bottom;
 }
+
+.bazel-version-indicator {
```

Comment:
This didn't work, do you mind I switch back to inlining this css?

Reply:
Sure, please do!

Reply:
Done!